parsimony/functions/multiblock/losses.py

```diff
@@ -1153,7 +1154,7 @@ def L(self, w, index):
         """
         # Any positive real number suffices, but a small one will give a larger
         # step in e.g. proximal gradient descent.
-        return np.sqrt(consts.TOLERANCE)  # 1.0
+        return 200 #np.sqrt(consts.TOLERANCE)  # 1.0
```

Comment:
I suppose 200 is as arbitrary as sqrt(TOLERANCE), but I think the latter has better theoretical justifications. I think this shouldn't be changed, or it should be overridden in the derived class.

Reply:
sqrt(TOLERANCE) results in a huge step size (~4472) so the algortihm didn't converge in my simulations

Reply:
Right. The loss is linear wrt. each variable, so the Lipschitz constant i zero, i.e. we can take an infinitely long step (to a function value of negative infinity). A value of 200 might work for a particular application, but will likely result in small steps in other. I suggest you change it back, and then in your own code create a derived class that overrides the L function with the value that works for you. Alternatively, let the constructor take a parameter (e.g., L or lipschitz_constant), such that a value of None (default) uses the default value of np.sqrt(consts.TOLERANCE) and any other positive float/int would be the value returned by the method L.

Reply:
Yes, what I did is directly pass the step size instead of L

Reply:
Ok, great! Can you also change back to returning np.sqrt(consts.TOLERANCE)?

Reply:
done